data/replaced_tags.txt

```diff
@@ -184,3 +184,7 @@ service=crossover : service=service
 
 denomination=latter_day_saints : denomination=mormon
 
+rental=city_bike : rental=bike
+rental=ebike : rental=bike
+rental=cargo_bike : rental=bike
+rental=e-bike : rental=bike
```

Comment:
a duplicate line

did you mean `city_bike;ebike` (5% of cases) ?

@vng colon-separated multiple values are not parsed automatically, right?

Reply:
yeap, confirmed there is no auto-parsing (only hardcoded for e.g. cuisines), so add it explicitly please

Reply:
ah ok 👍 i thought it'd be auto-parsed :)